globalsearch/utils/check_utils.py

```diff
@@ -0,0 +1,59 @@
+import flask
+
+
+def check_dict_has_none_values(dictionary):
+    has_none_value = False
+    for key, value in dictionary.items():
+        if value is None:
+            has_none_value = True
+
+    return has_none_value
+
+
+def check_enqueue_time_is_present(entity):
+    if not check_existence_and_non_null(entity, "enqueue_time"):
+        print(f"The enqueue time is missing or was null in update: {entity}.")
+        flask.abort(flask.make_response(flask.jsonify(error={
+            "code": "BadRequest",
+            "message": f"The enqueue time is missing or was null in update: {entity}."
+        }), 400))
+
+
+def check_existence_and_non_null(dictionary, key):
+    return key in dictionary and dictionary.get(key)
+
+
+def check_existence(dictionary, key):
+    return key in dictionary
+
+
+def check_all_values_are_non_null(entity):
+    has_none_values = check_dict_has_none_values(entity)
+
+    if has_none_values:
+        print(f"Update with null values was rejected")
+        flask.abort(flask.make_response(flask.jsonify(error={
+            "code": "NullValuesNotAllowed",
+            "message": "The request contained null values which are not allowed."
+        }), 400))
+
+    return
```

Comment:
Sometimes you explicitly return nothing, sometimes you don't. See line 19. Why?

Reply:
I think PyCharm generated the return when I extracted the methods. I got rid of them. Thanks for the note!